nl/guides/for-students/login-and-settings/README.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Aanmelden & gebruikersinstellingen
+description: "Tutorial: aanmelden en gebruikersinstellingen"
+---
+
+# Aanmelden & gebruikersinstellingen
+> Alle informatie zodat je als student een een gebruikersaccount door Dodona kan laten aanmaken en die kan personaliseren.
+
+## Inhoudstafel
+[[toc]]
+
+## Selecteren van de taal
+
+Als taal voor Dodona kan je kiezen tussen Nederlands of Engels. Gebruik hiervoor het taalmenu aan de rechterkant van de navigatiebalk (bovenaan elke pagina).
+
+![image](./choose_language.png)
+
+Als je een taal selecteert terwijl je aangemeld bent dan wordt die ook als voorkeurstaal ingesteld in je [gebruikersprofiel](). Op die manier wordt bij aanmelden automatisch je voorkeurstaal geselecteerd.
+
+::: tip Notitie
+
+De taal waarin vaste componenten van Dodona (menubalken, cursusstructuur, mededelingen, ...) worden weergegeven, valt volledig onder de controle van het platform. Deze componenten worden altijd weergegeven in de geselecteerde taal.
+
+De naam en beschrijving van een [cursus]() liggen vast, net zoals namen en beschrijvingen van [oefeningenreeksen]() binnen een cursus. Deze componenten zijn niet afhankelijk van de selectie van de taal. De taal van deze componenten valt volledig onder de controle van de cursusbeheerders die de cursus hebben opgesteld.
```

Comment:
cursus and oefeningenreeks are missing links

Reply:
This should be fixed after merging this branch and [teacher-docs](https://github.com/dodona-edu/dodona-edu.github.io/pull/73) since I have to link to pages from that branch. @winniederidder and I have made [issue #74 ](https://github.com/dodona-edu/dodona-edu.github.io/issues/74) to not forget this.

Reply:
In the end, I decided to remove all of the empty links since they were all supposed to link to the teacher guide that in my opinion is not meant for students to read. @bmesuere I would like to know If you agree with this decision.